redhat/nodes/insert-rolls.xml

```diff
@@ -33,6 +33,9 @@ generator.parse('/tmp/rolls.xml')
 
 found = []
 for name, version, release, arch, url, diskid in generator.rolls:
+	if url == 'http://127.0.0.1/mnt/cdrom/':
```

Comment:
Please leave a descriptive comment for future travelers about why this is here. If I was encountering this for the first time unaware of the context, I would be very suspicious of this seemingly brittle code.